__tests__/app.test.js

```diff
@@ -0,0 +1,51 @@
+const request = require('supertest');
+const app = require('../app')
+const seed = require('../db/seeds/seed.js');
+const testData = require('../db/data/test-data/');
+const connection = require('../db/connection')
+
+    describe('app', () => {
+        
+        beforeEach(() => seed(testData));
+        afterAll(()=> {
+            return connection.end()})
+        
+        describe('GET /api/topics', () => {
+            test('200: GET responds with an array of topics', () => {
+                return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then(( {body} ) => {
+                    const {topics} = body
+                   expect(topics.length).toBe(3)
+                    expect(topics).toBeInstanceOf(Array)
+                })
+            })
+            it('200: GET responds with an array of topics with correct properties', () => {
+                return request(app)
+                .get('/api/topics')
+                .expect(200)
+                .then(( {body} ) => {
+                    const {topics} = body
+                    topics.forEach((topic) => {
```

Comment:
I would add another expect here to check the length is what you would expect also.